Sources/PayPalWebPayments/PayPalVaultRequest.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+/// A request to vault a PayPal payment method
+public struct PayPalVaultRequest {
+
+    /// PayPal URL for the approval web page
+    public let url: URL
+
+    /// ID for the setup token associated with the vault
+    public let setupTokenID: String
+
+    /// Creates an instance of a PayPal vault request
+    /// - Parameters:
+    ///    - url: The PayPal URL for the approval web page
+    ///    - setupTokenID: An ID for the setup token to update
```

Comment:
The docstrings here vs above are different. Are we able to align the two?

Reply:
Great catch!

Reply:
Addressed here: 743b6afcafd0506de30da7eb932114a50df07b55